index.js

```diff
@@ -344,6 +345,45 @@ app.post('/api/doorTest', async(req, res) => {
     await handleSensorRequest(locationid)
 });
 
+app.post('/api/door', async(request, response) => {
+    const coreId = request.body.coreid
+    const locationid = db.getLocationFromCoreId(coreId)
```

Comment:
Hmmm, was this smart enough to give you just the `locationid` back from the `rows[0]` in `db.getLocationFromCoreId`? I'm surprised.

Reply:
Didn't test this, you have a good point. I will modify the function to return the `location `row, and then select the `locationid `column using `location.locationid`